Fix inaccuracy in HTMLTableRowElement doc (#11832)
mdn/content: files/en-us/web/api/htmltablerowelement/index.md

_Inherits methods from its parent, {{domxref("HTMLElement")}}_.

- {{domxref("HTMLTableRowElement.deleteCell()", "deleteCell(index)")}}

>   - : Removes the cell corresponding to `index`. If `index` is `-1`, the last cell of the row is removed; if `index` is less than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.

Reviewer: ```suggestion
  - : Removes the cell corresponding to `index`. If `index` is `-1`, the last cell of the row is removed. If `index` is less than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```

>   - : Removes the cell corresponding to the `index` given in parameter. If the `index` value is `-1` the last cell of the row is removed; if it is smaller than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.

Reviewer: ```suggestion
  - : Removes the cell corresponding to `index`. If `index` is `-1`, the last cell of the row is removed; if `index` is smaller than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```

Author: I wholeheartedly agree with your modification, but you would need to apply those throughout the other pages of the documentation for HTMLTableElement and others as well.

- {{domxref("HTMLTableRowElement.insertCell()", "insertCell(index)")}}

>   - : Returns an {{DOMxRef("HTMLTableCellElement")}} representing a new cell of the row. It inserts it in the cells collection immediately before the given `index` position in the row. If the `index` is `-1`, the new cell is appended to the collection. If the `index` is smaller than `-1` or greater than the number of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.

Reviewer: ```suggestion
  - : Returns an {{DOMxRef("HTMLTableCellElement")}} representing a new cell of the row. The cell is inserted in the collection of cells immediately before the given `index` position in the row. If `index` is `-1`, the new cell is appended to the collection. If `index` is less than `-1` or greater than the number of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```

Author: Of course, I agree and like I mentioned previously, those modifications should be applied to other parts of the table documentations for consistency.

_Inherits methods from its parent, {{domxref("HTMLElement")}}_.

> - {{domxref("HTMLTableRowElement.deleteCell()")}}

Reviewer: ```suggestion
- {{domxref("HTMLTableRowElement.deleteCell()", "deleteCell(index)")}}
```

  - : Removes the cell corresponding to the `index` given in parameter. If the `index` value is `-1` the last cell of the row is removed; if it is smaller than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.

> - {{domxref("HTMLTableRowElement.insertCell()")}}

Reviewer: ```suggestion
- {{domxref("HTMLTableRowElement.insertCell()", "insertCell(index)")}}
```
